### ISSUE TITLE
Vim theme?

### ISSUE THREAD

**Issue by benstokoe** [arcticicestudio/nord#7]
Hey,

What has happened to the vim theme? The repository is empty

**Comment by arcticicestudio** [comment]
Hi Ben,

the port project for Vim has no commits/code, I only created the empty repo to signal that it'll be populated with code in the future 😄 
I'm very busy on my job and other TODOs at the moment and haven't started with the project yet.

I also have to "go deeper" into the theming of Vim since there is a lot to consider like support for popular plugins. The project should **not** only consist of some lines defining the color palette, it should cover a large number of plugins and of course provide the **consistent and uniform coloring across programming languages** like all other Nord projects do.
In short words: quality needs some time 😃 

I'll make sure to notify you as soon as the repo comes alive.

**Comment by kamil-stachowski** [comment]
In the meantime, you might want to see my version at https://github.com/kamwitsta/nordisk. It's not strictly speaking a port but the overall aesthetic impression is quite similar.

**Comment by benstokoe** [comment]
@arcticicestudio ah I see! I didn't know that haha. I thought it had been deleted by accident or something! Nice one!

@kamwitsta that's what I currently have installed but in Javascript I'm getting things in yellow and not the blue colour etc, and the colours are off. Not sure if it's because I have the nord iTerm theme or if it doesn't support JS?

**Comment by kamil-stachowski** [comment]
@benstokoe JS falls under the general rules, I haven't defined anything specifically for it. I tried a few different bits of code now but did not see any yellow. Could you post the parts where you got the wrong colours?

**Comment by arcticicestudio** [comment]
The Nord iTerm2 project only defines the terminal color palette. Vim in turn defines which color should be used for which entity of a programming language.

That's what I meant with "**consistent and uniform coloring across programming languages**" so that parts, where most languages are equal (classes, types, operators, ...), are colored correct.

In two weeks my vacation starts and I'll have some time to relax and spent time on my project TODOs so I'll start the Vim project ;p

**Comment by benstokoe** [comment]
@kamwitsta this is what my vim looks like with a javascript file
![screen shot 2016-12-08 at 09 25 09](https://cloud.githubusercontent.com/assets/784093/21004448/496d45c2-bd28-11e6-872a-85b89d56c4bc.png)

**Comment by arcticicestudio** [comment]
You should create a issue in the Nordisk repository as it is related to his Vim theme.
I'll close this one here and notify you as soon as I start working on the Nord Vim theme.

**Comment by kamil-stachowski** [comment]
@benstokoe I did a test and it seems that the act of changing the iTerm palette overrides Vim. If you quit Vim, change the iTerm palette, quit iTerm, and then launch iTerm and Vim again, everything is as it should be. Let me know if this doesn't help.

**Comment by arcticicestudio** [comment]
@benstokoe @kamwitsta I've just released version [`0.1.0`](https://github.com/arcticicestudio/nord-vim/releases/tag/v0.1.0) of my new Nord Vim port project.
I'm happy for every feedback.
Ah, and happy christmas everyone 🎅
